src/stored_map.h

```diff
@@ -1,7 +1,7 @@
 #ifndef STORED_MAP_H
 #define STORED_MAP_H
 
-#include "tcumap.h"
+#include "../lib/core/tcumap.h"
```

Comment:
leave as just 'tcumap'h', pio automatically throws all library header files into the compile folder at build time. I don't know if specifying a relative directory might break something